collection-spec/examples/landsat-collection.json

```diff
@@ -1,10 +1,14 @@
 {
+    "stac_version": "0.7.0",
+    "stac_extensions": [
```

Comment:
```
"stac_extensions": {
        "custom_extension_1": "https://....",
        "eo": "0.0.1",
        "cube": "0.0.1"
    }
```

Reply:
See my comment below. I'm pretty sure we settled on arrays as the version information is basically included in the URL as the referenced schema is versioned.